### ISSUE TITLE
Cookie banner consent isn't displayed on boosted.orange.com

### ISSUE THREAD

**Issue by julien-deramond** [Orange-OpenSource/Orange-Boosted-Bootstrap#1382]
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

In the logs:
```
Uncaught TypeError: document.getElementById(...) is null
    color https://boosted.orange.com/docs/5.2/assets/js/docs.min.js:6
    color https://boosted.orange.com/docs/5.2/assets/js/docs.min.js:6
    addService https://boosted.orange.com/docs/5.2/assets/js/docs.min.js:6
    load https://boosted.orange.com/docs/5.2/assets/js/docs.min.js:6
    addScript https://boosted.orange.com/docs/5.2/assets/js/docs.min.js:6
    addInternalScript https://boosted.orange.com/docs/5.2/assets/js/docs.min.js:6
    load https://boosted.orange.com/docs/5.2/assets/js/docs.min.js:6
    onload https://boosted.orange.com/docs/5.2/assets/js/docs.min.js:6
docs.min.js:6:52024
```

### Reduced test cases

Simply go to https://boosted.orange.com and open the console

### What operating system(s) are you seeing the problem on?

macOS

### What browser(s) are you seeing the problem on?

Chrome, Firefox

### What version of Boosted are you using?

v5.2.0-beta1

**Comment by julien-deramond** [comment]
Should be fixed by #1389. Let's wait for the release of v5.2.0 before closing this issue.
Be careful with https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1390

**Comment by julien-deramond** [comment]
Closing it since the banner is now visible with the v5.2.0 release
